apps/dashboard/config/configuration_singleton.rb

```diff
@@ -397,6 +397,17 @@ def open_apps_in_new_window?
     end
   end
 
+  # How many days before a Session record is considered old and ready to delete
+  def ood_bc_card_time
+    ood_bc_card_time = ENV['OOD_BC_CARD_TIME']
+    return 7 if ood_bc_card_time.blank?
+
+    ood_bc_card_time_int = Integer(ood_bc_card_time)
+    (ood_bc_card_time_int < 0) ? 0 : ood_bc_card_time_int
+  rescue ArgumentError
+    7
```

Comment:
Yeah that's not a problem at all. 

So the regex is slightly off. It will pass the cases like `foo12` where ruby will just return 0. So we instead need to check if the string starts with an integer. I this this regex works well: `/^([+-]\d+|\d+)/`. In this we are checking that the string either begins with an integer  or a +/- followed by an integer.

Reply:
That'll work just fine!